torch/csrc/jit/codegen/cuda/test/test_gpu_indexing_ops.cpp

```diff
@@ -131,6 +131,642 @@ TEST_F(NVFuserTest, FusionSelectOpPersistent_CUDA) {
       &fusion, cg_outputs, {t0, idx}, {t7, t8, t9}, __LINE__, __FILE__);
 }
 
+TEST_F(NVFuserTest, FusionIndexSelectSimple_CUDA) {
+  Fusion fusion;
+  FusionGuard fg(&fusion);
+  // dimensionality of the problem
+  int nDims = 2;
+  int nElem = 1023;
+  int nElem_select = nElem + 115;
+  int nFeat = 128;
+
+  // Set up your input tensor views
+  TensorView* tv0 = makeContigTensor(nDims);
+  TensorView* tv_idx = makeContigTensor(1, DataType::Int);
+
+  // Register your inputs
+  fusion.addInput(tv0);
+  fusion.addInput(tv_idx);
+
+  TensorView* tv_sel = index_select(tv0, 0, tv_idx);
+  // Register your outputs
+  fusion.addOutput(tv_sel);
+
+  auto options = at::TensorOptions().dtype(at::kFloat).device(at::kCUDA, 0);
+  at::Tensor input0 = at::randn({nElem, nFeat}, options); // lookup
+  std::vector<int64_t> storage(nElem_select);
+  for (int i = 0; i < nElem_select; ++i) {
+    storage[i] = std::rand() % nElem;
```

Comment:
Let's make this deterministic. Using rand is fine, but this means selected indices are non deterministic, which make debugging harder. Please deterministically initialize a random seed.

It may also be a good idea to have multiple runs with different index values, but they should be deterministically generated. So, for example:

```
for (int i = 0; i < 10; ++i) {
   int rand_seed = i;
   // initialize rand with rand_seed
   std::vector<int64_t> storage(nElem_select);
   for (int i = 0; i < nElem_select; ++i) {
     storage[i] = std::rand() % nElem;
      ...
}
```

Reply:
Done, added seed into all UTs and added multiple runs on some UTs.